controllers/grid/catalogEntry/form/PublicationFormatForm.inc.php

```diff
@@ -188,6 +205,36 @@ function execute(...$functionParams) {
 			SubmissionLog::logEvent(Application::get()->getRequest(), $this->getMonograph(), SUBMISSION_LOG_PUBLICATION_FORMAT_CREATE, 'submission.event.publicationFormatCreated', array('formatName' => $publicationFormat->getLocalizedName()));
 		}
 
+
+		// Save ISBN identification code
```

Comment:
I think you can remove this comment since the other comments are sufficient.

Can you move all of this added code up above the `SubmissionLog::logEvent()`? The event should come after the action is completed. Won't matter too much now, but as we move to decouple events more, the order of actions will matter more. (Imagine hooking into this event: the person hooking in would not have the latest value for ISBNs.)

Reply:
ok